tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -349,3 +350,61 @@ def test_get_manifest(self, run_command: MagicMock):
 
         run_command.side_effect = ReleaseNotFoundException()
         assert helm_wrapper.get_manifest("test-release", "test-namespace") == ()
+
+    def test_should_call_run_command_method_when_helm_template_with_optional_args(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+            values={"commandLine": "test"},
+            flags=HelmTemplateFlags(
+                api_version="2.1.1",
+                ca_file="a_file.ca",
+                cert_file="a_file.pem",
+            ),
+        )
+        run_command.assert_called_once_with(
+            command=[
+                "helm",
+                "template",
+                "test-release",
+                "bakdata-streams-bootstrap/streams-app",
+                "--values",
+                "values.yaml",
+                "--api-versions",
+                "2.1.1",
+                "--ca-file",
+                "a_file.ca",
+                "--cert-file",
+                "a_file.pem",
+            ],
+        )
+
+    def test_should_call_run_command_method_when_helm_template_without_optional_args(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+            values={"commandLine": "test"},
+            flags=HelmTemplateFlags(
+                api_version="",
+                ca_file="",
+                cert_file="",
```

Comment:
I believe these should be `None` since you changed the default

Reply:
in this case `HelmTemplateFlags()` without any arguments would make sense

Reply:
cool, changed it